README.md

```diff
@@ -1,159 +1,64 @@
-
-### Run
-
-1. From the project root, execute the `runthedocs.sh` script:
-
-    ```bash
-    cd documentation
-    ./runthedocs.sh
-    ```
-
-    **Note::** This will occupy the terminal in which you invoke the script, providing output when files are changed or loaded. Use **CTRL** + **C** to exit.
-
-    You should see:
-
-    ```bash
-    Creating the Docker container...
-    Container built and running. 👍👍
-    Building the latest docs...
-    Detected new or updated files
-    Generating: 100% (1035 sources / 0.06 seconds)
-    Converting: 100% (1122 sources / 8.04 seconds)
-    Formatting: 100% (1122 sources / 2.73 seconds)
-    Processing completed in 12.20 seconds
-    Running "watch" task
-    Waiting...
-    ```
-
-    Once running, you can view the local dev environment by visiting `http://localhost:8000/docs` in your browser.
-
-    The Docker image will mount the `source` directory from the project root on your computer, and rebuild when you save changes:
+   ```bash
+   npm install -g gatsby-cli
+   ```
 
-    ```bash
-    /documentation/source/_docs/getting-support.md was changed. Building...
-    Detected new or updated files
-    Generating: 100% (1035 sources / 0.04 seconds)
-    Converting: 100% (1122 sources / 5.73 seconds)
-    Formatting: 100% (1122 sources / 2.01 seconds)
-    Processing completed in 9.05 seconds
-    ```
-
-1. When you're done, run:
-
-    ```bash
-    docker container stop pantheon-docs
-    ```
-
-### Update The Docs
+### Get the Code
 
-The `pantheon-docs` container mounts the `documentation/source` directory from your computer, so the image remains evergreen as long as the toolset is unchanged. To update your local project with the latest version of the docs, you need only pull them from git:
+Fork and clone this repository.
 
 ```bash
-git pull origin
+git clone https://github.com/pantheon-systems/documentation.git
 ```
 
-### Update The App
-
-
-
-1. To update the documentation build locally run:
-
-    ```bash
-    ./updater.sh
-    ```
-
-### Testing
-
-We include several tools to test that new content doesn't break the documentation. Most of these tests are performed automatically by our continuous integration service, but pull requests created from external contributors aren't included in CI tests. If you want to manually test your branch, you can execute the following tests within the Docker container.
-
-#### Merge Conflicts
-
-To check for merge conflict messages accidentally commited into the docs, run the `merge_conflicts.sh` script:
+Or
 
 ```bash
-docker exec pantheon-docs scripts/merge_conflicts.sh
-Merge conflict:pass
+git@github.com:pantheon-systems/documentation.git
 ```
-#### [HTMLProofer](https://github.com/gjtorikian/html-proofer)
 
-To make sure all relative links (like `/docs/support/`) are correct, run
+### Install
 
-```bash
-docker exec pantheon-docs htmlproofer --assume-extension  ./output_dev/ --disable-external true
+```
+cd documentation/gatsby
+npm install
 ```
 
```

Comment:
I've encountered errors at this step when I don't include a GitHub token. Should we include those steps here? (md is approximate - not visually tested yet)

```
1) Gatsby needs a GitHub token to build. To create a GitHub token:
    1) Log in to GitHub and go to https://github.com/settings/tokens
    1) Click Generate new token
    1) Name it something and click the public_repo checkbox, then the Generate Token button at the bottom
    1) Copy the token
    1) Create or edit the .env.development file in the gatsby directory and add it in (replace $TOKENHASH ):
        `GITHUB_API=$TOKENHASH`
```

Reply:
Yes, I forgot about that - I'll update.